apps/site/components/Common/Searchbox/index.tsx

```diff
@@ -1,58 +1,63 @@
 'use client';
 
-import { MagnifyingGlassIcon } from '@heroicons/react/24/solid';
-import { SearchRoot, ChatRoot, Modal } from '@orama/ui/components';
+import ChatTrigger from '@node-core/ui-components/Common/Search/Chat/Trigger';
+import SearchModal from '@node-core/ui-components/Common/Search/Modal';
+import SearchResults from '@node-core/ui-components/Common/Search/Results';
+import SearchSuggestions from '@node-core/ui-components/Common/Search/Suggestions';
 import { useTranslations } from 'next-intl';
-import type { FC } from 'react';
+import { useState, type FC } from 'react';
 
-import '@orama/ui/styles.css';
-import { SearchboxProvider } from '#site/providers/searchboxProvider';
+import { DEFAULT_ORAMA_QUERY_PARAMS } from '#site/next.constants.mjs';
 
+import type { Document } from './DocumentLink';
+import { Footer } from './Footer';
 import styles from './index.module.css';
```

Comment:
Can we please have the style import be the last thing in the bottom? Usually good to ensure other component styles don't get loades after this (even if technically should be separated)